packages/frontend/app/login/components/login-form.tsx

```diff
@@ -38,6 +38,9 @@ export function LoginForm() {
     await dispatch(loginUserThunk({ ...data, router }));
   };
 
+  if (sessionStorage?.getItem("sessionToken"))
+    return router.replace("/dashboard");
```

Comment:
You shouldn't do the early returns because sessionStorage and router rely on the component being on the client. This would need to be done in a useEffect because that is guaranteed to be client side. If you do it bare like that it will break the build (hence why CI must be failing).

Reply:
ok yeah that makes sense... thought this was a bit janky 😅